portfolio/src/main/webapp/script.js

```diff
@@ -17,12 +17,37 @@
  */
 function addRandomGreeting() {
   const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!',
+			'Quote1', 'Quote2', 'Quote3'];
 
   // Pick a random greeting.
   const greeting = greetings[Math.floor(Math.random() * greetings.length)];
 
   // Add it to the page.
   const greetingContainer = document.getElementById('greeting-container');
+	
   greetingContainer.innerText = greeting;
 }
+
+/**
+ * Fetches stats from the servers and adds them to the DOM.
+ */
+function getComment() {
+  fetch('/data').then(response => response.json()).then((stats) => {
+    // stats is an object, not a string, so we have to
+    // reference its fields to create HTML content
+
+    const statsListElement = document.getElementById('comment-container');
+    statsListElement.innerHTML = '';
+    var i;
+    for (i = 0; i < stats.length; i++) {
```

Comment:
Can be `for (var i = 0; i`
Also please check comments in previous PRs

Reply:
alright, will change according, didn't know I could do for loop like that